### PR TITLE
block_multifunction:disable pcie-root-port device for ppc

### PR DESCRIPTION
**Pull request by maxujun**
ID:2118860
Signed-off-by: Xujun Ma <xuma@redhat.com>

### CODE REVIEW

**Comment by maxujun** [comment]
@qingwangrh @MiriamDeng Could you help review?

**Comment by qingwangrh** [comment]
Passed python ConfigTest.py --testcase=block_multifunction.default.q35 --iothread_scheme=roundrobin --nr_iothreads=2 --platform=x86_64 --guestname=RHEL.9.0.0 --platform=x86_64 --driveformat=virtio_scsi --imageformat=qcow2 --machines=q35   --nrepeat=5  --firmware=default_bios --clone=no

LGTM

**Comment by MiriamDeng** [comment]
LGTM

**Comment by fbq815** [comment]
After research, I think we don't support this task on s390x:
[root@l42 ~]# /usr/libexec/qemu-kvm -device virtio-scsi-ccw,? | grep -i multi
[root@l42 ~]#

File: qemu/tests/cfg/block_multifunction.cfg [hunk 1]
```diff
@@ -17,4 +17,5 @@
             type = block_multifunction_scale
             only virtio_blk
             start_vm = yes
-            pcie_extra_root_port = 3
+            x86_64,i386:
```
Comment (PaulYuuu) [inline qemu/tests/cfg/block_multifunction.cfg:20]: on x86, only q35 supports pcie.

Comment (maxujun) [reply qemu/tests/cfg/block_multifunction.cfg:20]: Updated.

**Comment by qingwangrh** [comment]
@menli820 @peixiu Please help to review .  Thanks

**Review by PaulYuuu — APPROVED** [review]

**Comment by peixiu** [comment]
Checked virtio-win vioscsi related cases, this change will not impact our current tests~

Acked-by: Peixiu Hou phou@redhat.com
Thanks~

**Comment by menli820** [comment]
The change has no influence on virtio-win viostor cases.

LGTM

**Comment by PaulYuuu** [comment]
Thanks all.

**Pull request merged**